### PR TITLE
App Mesh Kubernetes guide update for v1.0.0

### PR DESCRIPTION
**Pull request by fawadkhaliq**
*Description of changes:*
App Mesh Kubernetes integration guide update for v1.0.0

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: doc_source/mesh-k8s-integration.md [hunk 1]
```diff
@@ -58,46 +58,38 @@ Install the integration components one time to each cluster that hosts pods that
...
+    --set serviceAccount.create=false \
+    --set serviceAccount.name=appmesh-manager
    ```
 
-1. Confirm that the controller version is `v0.3.0` or later\. You can view the [change log](https://github.com/aws/aws-app-mesh-controller-for-k8s/blob/master/CHANGELOG.md) on GitHub\.
+1. Confirm that the controller version is `v1.0.0` or later\. You can view the [change log](https://github.com/aws/aws-app-mesh-controller-for-k8s/blob/master/CHANGELOG.md) on GitHub\.
 
    ```
-   kubectl get deployment -n appmesh-system appmesh-controller -o json  | jq -r ".spec.template.spec.containers[].image" | cut -f2 -d ':'
+   kubectl get deployment -n appmesh-system appmesh-manager -o json  | jq -r ".spec.template.spec.containers[].image" | cut -f2 -d ':'
```
Comment (achevuru) [inline doc_source/mesh-k8s-integration.md:90]: Deployment name should be "appmesh-controller-manager"

File: doc_source/mesh-k8s-integration.md [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
-+  **App Mesh sidecar injector for Kubernetes** – The injector installs as a webhook and injects the following containers into Kubernetes pods that are running in specific, labeled namespaces\. To learn more or contribute, see the [GitHub project](https://github.com/aws/aws-app-mesh-inject)\.
-  + **App Mesh Envoy proxy** –Envoy uses the configuration defined in the App Mesh control plane to determine where to send your application traffic\.\. 
-  + **App Mesh proxy route manager **– The route manager sets up a pod’s network namespace with `iptables` rules that route ingress and egress traffic through Envoy\.
+When you integrate AWS App Mesh with Kubernetes, you manage App Mesh resources, such as meshs, virtual services, virtual nodes and virtual routers, through Kubernetes\. You also automatically add the App Mesh sidecar container images to Kubernetes pod specifications\. This tutorial guides you through the installation of the following open source components that enable this integration:
```
Comment (achevuru) [inline doc_source/mesh-k8s-integration.md]: Minor typo .. "meshs" -> "meshes"

File: doc_source/mesh-k8s-integration.md [hunk 3]
```diff
@@ -1,10 +1,10 @@
...
++  **App Mesh sidecar injector for Kubernetes** – The injector installs as a webhook and injects the following containers into Kubernetes pods that are running in specific, labeled namespaces\. The sidecar injector runs inside the App Mesh manager.
+    + **App Mesh Envoy proxy** –Envoy uses the configuration defined in the App Mesh control plane to determine where to send your application traffic\.\.
+    + **App Mesh proxy route manager **– The route manager sets up a pod’s network namespace with `iptables` rules that route ingress and egress traffic through Envoy\. This runs as a Kubernetes init container inside a pod
```
Comment (achevuru) [inline doc_source/mesh-k8s-integration.md]: Might not be accurate to say proxy init container "sets up a pod’s network namespace". I think we can just say "Updates 'iptables' rules in pod's network namespace to route ingress and egress pod traffic through Envoy"

File: doc_source/mesh-k8s-integration.md (hunk 2 above)
Comment (fawadkhaliq) [reply doc_source/mesh-k8s-integration.md]: Already fixed

File: doc_source/mesh-k8s-integration.md (hunk 1 above)
Comment (fawadkhaliq) [reply doc_source/mesh-k8s-integration.md:90]: Actually it is appmesh-manager

https://github.com/aws/eks-charts/tree/master/stable/appmesh-manager

File: doc_source/mesh-k8s-integration.md (hunk 2 above)
Comment (achevuru) [reply doc_source/mesh-k8s-integration.md]: General Comment: I think it is also better to add that deleting a Mesh will not automatically trigger delete of VirtualNodes/VirtualServices/VirtualRouters under that mesh. Other AppMesh K8S resources should be deleted prior to attempting a mesh delete. This is different to how the old controller behaves, so we should probably mention it.

Comment (fawadkhaliq) [reply doc_source/mesh-k8s-integration.md]: Good point on the mesh delete. I'll add it

File: doc_source/mesh-k8s-integration.md (hunk 3 above)
Comment (fawadkhaliq) [reply doc_source/mesh-k8s-integration.md]: Hmm, why would it not be accurate to call it an init container. Is it an init container, no?

Comment (achevuru) [reply doc_source/mesh-k8s-integration.md]: No, the comment was about - "sets up a pod’s network namespace". Init container doesn't do that right, it just updates iptable rules

Comment (fawadkhaliq) [reply doc_source/mesh-k8s-integration.md]: Ah, I see the confusion. Fair point. I'll adjust the wording

**Comment by fawadkhaliq** [comment]
Closing this. The changes have been incorporated by @jimdial-aws already

**Pull request closed without merging**